SwiftLeeds/Views/Sponsors/SponsorsView.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  SponsorsView.swift
+//  SwiftLeeds
+//
+//  Created by Muralidharan Kathiresan on 25/06/23.
+//
+
+import SwiftUI
+
+struct SponsorsView: View {
+    @Environment(\.openURL) var openURL
+    @StateObject private var viewModel = SponsorsViewModel()
+    
+    var body: some View {
+        List {
+            ForEach(viewModel.sections) { section in
+                switch section.type {
+                case .platinum:
+                    Section(header: sectionHeader(for: section.type)) {
+                        ForEach(section.sponsors) { sponsor in
+                            contentTile(for: sponsor)
+                                .listRowBackground(Color.clear)
+                        }
+                    }
+                case .gold:
+                    Section(header: sectionHeader(for: section.type)) {
+                        ForEach(section.sponsors) { sponsor in
+                            contentTile(for: sponsor)
+                                .listRowBackground(Color.clear)
```

Comment:
A quick tip around this one - you can also remove the padding by doing `.listRowInsets(EdgeInsets())` which will then remove the extra padding, and let your content show as the same size as a normal list row 😄

Reply:
Addressed <img src = "https://github.com/SwiftLeeds/swiftleeds-ios/assets/7802406/67edcd0b-c551-4f90-b02e-726623f7d8a3" height = 550>